modules/advanced-icon/includes/frontend.css.php

```diff
@@ -8,6 +8,22 @@
 ?>
 <?php
 
+// Ensure $settings is defined and initialized
+if (!isset($settings)) {
+    $settings = new stdClass(); // Create an empty object to avoid undefined errors
+}
+
+// Ensure $id is defined and initialized
+if (!isset($id)) {
```

Comment:
a different $id name to be used such as $module_id etc, as $id is reserved

Reply:
we are using $id as the code has only $id and the PHPStan error we were receiving was -> $id was not defined, so we only needed to define it properly.